pkgs/development/libraries/microsoft_gsl/default.nix

```diff
@@ -1,23 +1,23 @@
-{ stdenv, fetchgit, cmake
+{ stdenv, fetchgit, catch, cmake
 }:
 
 let
   nativeBuild = stdenv.hostPlatform == stdenv.buildPlatform;
 in
 stdenv.mkDerivation rec {
   name = "microsoft_gsl-${version}";
-  version = "2017-02-13";
+  version = "2.0.0";
 
   src = fetchgit {
```

Comment:
Please use the source archive instead of `fetchgit` (from a cursory glance it does not appear to me that `fetchgit` will do so by itself).

Reply:
Good catch!